### PR TITLE
Release 1.10.0

### PR DESCRIPTION
**Pull request by nikitin-da**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 StorIO Change Log
 ==========
 
+## Version 1.9.2
+
+_2016_07_22_
+
+* Find type mapping among interfaces recursively. Pluggable typemapping!
+* Default scheduler handling for StorIOSQLite
```
Comment (artem-zinnatullin) [inline CHANGELOG.md]: Just: Default scheduler for StorIOSQLite?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+* Default scheduler handling for StorIOContentResolver
```
Comment (artem-zinnatullin) [inline CHANGELOG.md]: Same as above

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,20 @@
 StorIO Change Log
 ==========
 
+## Version 1.9.2
```
Comment (artem-zinnatullin) [inline CHANGELOG.md]: I think these change deserve `1.10.0`!

**Comment by nikitin-da** [comment]
@artem-zinnatullin done!

**Pull request merged**